examples/Console/Program.cs

```diff
@@ -163,6 +163,12 @@ internal class LogsOptions
 
     [Option('p', "protocol", HelpText = "Transport protocol used by OTLP exporter. Supported values: grpc and http/protobuf. Only applicable if Exporter is OTLP", Default = "grpc")]
     public string Protocol { get; set; }
+
+    [Option("processorType", Default = "batch", HelpText = "export processor type. Supported values: simple and batch", Required = false)]
```

Comment:
I think we should be consistent here with the options for Traces. Either remove these options for Logs to keep the example simple or also add this for Traces.

 https://github.com/open-telemetry/opentelemetry-dotnet/blob/9e919044943f39f0dc2b2572be48b7f6fa12730b/examples/Console/Program.cs#L145-L153

Reply:
We need to add these for traces, once we resolve https://github.com/open-telemetry/opentelemetry-dotnet/issues/4527